openLuup/init.lua

```diff
@@ -1,6 +1,6 @@
 local ABOUT = {
   NAME          = "openLuup.init",
-  VERSION       = "2018.04.29",
```

Comment:
Actually, nothing in the init module should be changed.

The changes made here have no effect, and, anyway, the parameter list of the luup.create_device() request should not be changed.